slack/client.go

```diff
@@ -108,14 +119,25 @@ func (c *Client) RunHistoricalBackfilling(ctx context.Context) error {
 
 // Export sends slack message to the additional service via callback
 func (c *Client) Export(cb func(m Message)) {
+	ticker := time.NewTicker(batchSendInterval)
 	for {
 		select {
+		case <-ticker.C:
+			log.Printf("sending batch of %d messages", len(c.batch))
+			c.mx.Lock()
+			for _, m := range c.batch {
+				cb(m)
+				messageOutCount.Inc()
```

Comment:
Can this `.Inc()` call also be moved out of `for` loop and be replaced with the following?
```
messageOutCount.Add(len(c.batch))
```

Reply:
updated